src/components/PopOver/index.js

```diff
@@ -135,17 +136,22 @@ class PopOver extends Component {
         ? topPosition
         : bottomPosition;
 
-    return {
-      x: Math.min(
-        Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
-        windowWidth - spaceFromEdge - width,
-        containerRight
-      ),
-      y:
-        bottomPosition + height + spaceFromEdge <=
+    const getXPosition = Math.min(
+      Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
+      windowWidth - spaceFromEdge - width,
+      containerRight
+    );
+
+    const getYPosition =
+      !preferTop &&
```

Comment:
I don't think that works. preferTop mist also validate that there is space on top to display the tooltip. Here we are only checking the bottom space if !preferTop.

Reply:
this seems to be working correctly in the example, see screenshot:
the calculatePosition, only really calculates the position of the tooltip relative to the parent elementm, but the actual display is handled in the tooltip component itself, which validates the top space: https://github.com/ticketmaster/aurora/blob/master/src/components/Tooltip/index.js#L118

![Screen Shot 2020-03-10 at 09 51 41](https://user-images.githubusercontent.com/52719588/76337591-d0816e00-62b4-11ea-8b59-26f881abcfe6.png)

![Screen Shot 2020-03-10 at 09 51 54](https://user-images.githubusercontent.com/52719588/76337604-d414f500-62b4-11ea-9fc5-9880bb498022.png)